fling/__init__.py

```diff
@@ -1,6 +1,6 @@
 __TITLE__ = 'Fling'
 __VERSION__ = 'v0.0.5'
 __DESCRIPTION__ = 'Federated Learning Framework'
-__AUTHOR__ = "BUAA-FLAIR"
-__AUTHOR_EMAIL__ = "liu_xuefeng@buaa.edu.cn"
+__AUTHOR__ = "kxzxvbk"
```

Comment:
The merge logic is not correct. It seems that the base version you use is outdated